tests/unit/compat_filter.js

```diff
@@ -0,0 +1,117 @@
+define('tests/unit/compat_filter',
+    ['tests/unit/helpers'],
+    function(helpers) {
+
+    var TEST_ENDPOINTS = ['category', 'recommended', 'installed', 'search'];
+
+    function testApiArgs(compatFilter, dev, device, limit, pro) {
```

Comment:
it's fine, but I'm not a super big fan of using positional arguments here. I'd prefer passing a litteral object so that we can call `testApiArgs(compatFilter, {dev: '', device: '', limit: 25, pro: undefined})`.